Client/mods/deathmatch/logic/CClientRenderElementManager.h

```diff
@@ -36,6 +37,7 @@ class CClientRenderElementManager
     CClientScreenSource*  CreateScreenSource(uint uiSizeX, uint uiSizeY);
     CClientWebBrowser*    CreateWebBrowser(uint uiSizeX, uint uiSizeY, bool bIsLocal, bool bTransparent);
     CClientVectorGraphic* CreateVectorGraphic(uint width, uint height);
+    CClientGif*           CreateGif(uint width,uint height);
```

Comment:
Formatting is kind of all over the place (for example, there should be a space after each comma for the parameters.

Are you using the clang-format included with the repo? Visual Studio _should_ pick this up and allow you to format the current working file with the default shortcut. (`Ctrl + K, Ctrl + D`).

You could also run [win-apply-clang-format.bat](https://github.com/multitheftauto/mtasa-blue/blob/master/utils/win-apply-clang-format.bat) but it's probably going to create a ton of unrelated formatting changes so I'd advise against it for now.

Reply:
fixed!